src/ui/_theme.scss

```diff
@@ -19,9 +19,9 @@
  * rhythm and spacing.
  */
 
-$document-font-size: 14px !default;
+$document-font-size: 16px !default;
 $document-line-height: 1.6 !default;
-$document-rhythm: 2rem !default;
+$document-rhythm: $document-font-size * $document-line-height !default;
```

Comment:
Wouldn't this be overridden by the $document-rhythm set in your rhythm function?